### DISCUSSION TITLE
Azure VM sizes with no local temporary disk - jakie macie przemyslenia?

### DISCUSSION THREAD

**Discussion by MariuszFerdyn** [AzureCommunityPL/group#137] [Ogólna]
Azure VM sizes with no local temporary disk.

Jakie macie przemyślenia - bo przeważnie przydaje się ten szybki dysk na chociażby SwpaFile nie mówiać już o TEMP DB.
Moge sobie wyobraziś serwery w których jest to zbędne - pamietam te dyskusje na temat SWAP i SQLa, ale tutaj i tak się przyda.

Na serwery typu RDP też raczej przydatne.

https://docs.microsoft.com/pl-pl/azure/virtual-machines/azure-vms-no-temp-disk

**Comment by eklime** [comment]
A na co to komu potrzebne?

**Comment by piotrgwiazda** [comment]
Chodzi o wybór. Jeśli tego nie potrzebuję to po co za to płacić? W GCP dysk
Local SSD się podpina na żądanie.
Wszystkie klastry Kubernetes, batch, HPC, różne bastion hosty, niektóre
klastry typu Spark/Flink/Dremio (chyba, że RocksDB używamy). Pewnie sam
Azure będzie ich używał dla PaaS, FaaS itd. Lokalne dyski są drogie i
nieporęczne.

czw., 5 lis 2020, 09:52 użytkownik Emil Wasilewski <notifications@github.com>
napisał:
